README.md

```diff
@@ -26,6 +26,7 @@ install_plugin Capistrano::SCM::Git
 to
 ```ruby
 require "capistrano/scm/git-with-submodules"
```

Comment:
Since `git-with-submodules` already does that it isn't technically required, but I will add it anyway in a moment.